src/main/java/com/yodle/vantage/component/dao/VersionDao.java

```diff
@@ -91,7 +91,8 @@ public boolean createResolvedImplicitDependency(Version version, Version dep, Co
         jdbcTemplate.update(
                 "MATCH (c:Component {name: {1}})<-[:VERSION_OF]-(v:Version {version:{2}})," +
                         "(c_new:Component {name: {4}})<-[:VERSION_OF]-(v_new:Version {version:{3}}) " +
-                        "CREATE UNIQUE (v)-[:DEPENDS_ON {profiles:{5}}]->(v_new)",
+                        "MERGE (v)-[r:DEPENDS_ON]->(v_new) " +
```

Comment:
General neo4j good practice:  when doing merging, merge in the smallest increments possible and set properties on things you might create as part of the merge separately.  The old code basically said 'if you don't have a depends_on relationship with these properties set, create one', meaning if a relationship existed but with different properties it would create an entirely new relationship.